### PR TITLE
Design improvements

### LINKED ISSUE #79: web reader nav improvements
remove upper drop shadow from settings menu
decrease / responsify the location / positioning at the bottom of the screen
inline the controls for larger screens

### PR DESCRIPTION
**Pull request by ricardoom**
fixes #79

### CODE REVIEW

**Comment by aslagle** [comment]
This happened when I clicked scrolling...
screen shot 2017-05-02 at 10 30 42 am

**Comment by aslagle** [comment]
Also the alignment of the font label message is a little off when you increase the size.
screen shot 2017-05-02 at 10 33 47 am

**Comment by ricardoom** [comment]
regarding the first screen shot, I'm not seeing that breaking of the controls to the next line...
regarding the second, i'll fix that. 
I'll pull this pr.

**Comment by ricardoom** [comment]
@aslagle I made the `.fontSizeLabel` fixed so we dont have such a jumpy experience when resizing the type size...

**Comment by ricardoom** [comment]
checks failing due to commented code, not sure how to modify those tests...

File: src/BookSettings.ts [hunk 1]
```diff
@@ -210,23 +210,23 @@ export default class BookSettings {
...
+    //         this.fontSizeButtons["decrease"].className = "decrease";
+    //     }
+    //
+    //     if (currentFontSizeIndex === this.fontSizes.length - 1) {
+    //         this.fontSizeButtons["increase"].className = "increase disabled";
+    //     } else {
+    //         this.fontSizeButtons["increase"].className = "increase";
+    //     }
+    //
+    //     this.fontSizeLabel.style.fontSize = this.selectedFontSize;
```
Comment (aslagle) [inline src/BookSettings.ts]: you took out a lot more code than you needed to. you could've removed only this last line that sets the font size.

**Comment by aslagle** [comment]
@ricardoom I think you could put back most of the code, and then delete the one test that checks if the font size label changes size.

**Comment by ricardoom** [comment]
ok. will do.

File: src/styles/sass/main.scss [hunk 2]
```diff
@@ -293,12 +293,15 @@ a {
   }
 }
 
+.contents-view {
+  overflow: auto;
+}
```
Comment (aslagle) [inline src/styles/sass/main.scss]: you have overflow:auto in here but overflow: scroll in _toc.scss?

File: src/BookSettings.ts [hunk 3]
```diff
@@ -225,7 +225,7 @@ export default class BookSettings {
             this.fontSizeButtons["increase"].className = "increase";
         }
 
-        this.fontSizeLabel.style.fontSize = this.selectedFontSize;
+        //this.fontSizeLabel.style.fontSize = this.selectedFontSize;
```
Comment (aslagle) [inline src/BookSettings.ts]: could you remove all the commented out lines?

Comment (ricardoom) [reply src/BookSettings.ts]: np

File: src/styles/sass/main.scss (hunk 2 above)
Comment (ricardoom) [reply src/styles/sass/main.scss]: ok. really we want that to get rid of this all together... so TOC has `overflow: scroll` and _settings has `overflow: hidden`

Comment (ricardoom) [reply src/styles/sass/main.scss]: derp. just double declared it as all...

File: test/BookSettingsTest.ts [hunk 4]
```diff
@@ -219,17 +219,17 @@ describe("BookSettings", () => {
 
             const decreaseButton = element.querySelector("button[class=decrease]") as HTMLButtonElement;
             const increaseButton = element.querySelector("button[class=increase]") as HTMLButtonElement;
-            const label = element.querySelector(".font-size-label") as HTMLLIElement;
+            //const label = element.querySelector(".font-size-label") as HTMLLIElement;
```
Comment (aslagle) [inline test/BookSettingsTest.ts]: could you remove the commented out lines in the test too?

Comment (ricardoom) [reply test/BookSettingsTest.ts]: deeerrrrrrppp. yes.

**Comment by ricardoom** [comment]
@aslagle can it be dug?

**Pull request merged**